flow/scripts/report_metrics.tcl

```diff
@@ -226,9 +226,9 @@ proc report_metrics { stage when {include_erc true} {include_clock_skew true} }
   }
 
   # TODO these only work to stdout, whereas we want to append to the $filename
```

Comment:
the comment would be obsolete with this change

Reply:
Will fix. I take it from the lack of other comments that there was no particular reason and that fixing it is just good gardening.